### PR TITLE
Fix/deposit modal

### PR DESCRIPTION
**Pull request by VonIobro**
fix issues found by @mateleshkavo 
 https://acebusters.slack.com/archives/C63DZMB45/p1508569279000079

### CODE REVIEW

File: app/components/Dashboard/Invest.js [hunk 1]
```diff
@@ -113,5 +115,7 @@ Invest.propTypes = {
   estimatePowerUp: PropTypes.func.isRequired,
   handlePowerDown: PropTypes.func.isRequired,
   estimatePowerDown: PropTypes.func.isRequired,
+  modalAdd: PropTypes.func.isRequired,
+  modalDismiss: PropTypes.func.isRequired,
```
Comment (sunify) [inline app/components/Dashboard/Invest.js]: No need modal actions for invest component (and container). Only place where it used — ExchangeDialog, but it's a container and it's binding modal actions by self

File: app/containers/Dashboard/index.js [hunk 2]
```diff
@@ -119,9 +120,13 @@ DashboardRoot.propTypes = {
...
 };
 
-const mapDispatchToProps = () => ({
-  setActiveTab,
-});
+function mapDispatchToProps() {
+  return {
+    setActiveTab,
+    modalDismiss,
+    modalAdd,
```
Comment (sunify) [inline app/containers/Dashboard/index.js]: Root dashboard container is the bad place for passing modal actions. It does not require it, so will be better to bind this actions in subcontainers (in Wallet for example).

File: app/components/Dashboard/Invest.js (hunk 1 above)
Comment (VonIobro) [reply app/components/Dashboard/Invest.js]: OK good catch.

File: app/containers/Dashboard/index.js (hunk 2 above)
Comment (VonIobro) [reply app/containers/Dashboard/index.js]: OK, will change, too

**Pull request merged**